crates/ironrdp-svc/src/lib.rs

```diff
@@ -281,7 +336,7 @@ macro_rules! impl_as_any {
     };
 }
 
-/// A set holding at most one [`StaticVirtualChannel`] trait object for any given static channel and
+/// A set holding at most one [`StaticVirtualChannel`] for any given static channel and
 /// its corresponding [`ChunkProcessor`].
```

Comment:
Since the `StaticVirtualChannel` is now holding both the `ChunkProcessor` and the `StaticVirtualChannelProcessor`, maybe the doc should be modified further, something like
```rust
/// A set holding at most one [`StaticVirtualChannel`] for any given type
/// implementing [`StaticVirtualChannelProcessor`]
```